### PR TITLE
PIPE2D-536: Correct header info for ReduceArc and BootstrapDetectorMap

### PR DESCRIPTION
**Pull request by hassanxp**
Added new method std_raw_md() to fix raw_md headers.
Added docstring for that above method
Added docstring for the previously existing and similar std_raw method.

### CODE REVIEW

File: python/lsst/obs/pfs/pfsMapper.py [hunk 1]
```diff
@@ -216,6 +226,22 @@ def std_raw(self, item, dataId):
...
+        This should be done by the CameraMapper base class, but it isn't yet (DM-23959).
+
+        Parameters
+        ----------
+        item : `lsst.daf.base.propertyContainer.propertyList.PropertyList`
+
+        dataId : `dict`
+            Dataset identifier
+        """
+        self.log.info(f"type of item is: {type(item)}")
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:241]: Remove debugging statement.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 2]
```diff
@@ -200,6 +200,16 @@ def std_raw(self, item, dataId):
         does not exist.
 
         See _shiftAmpPixels() for the implementation.
+
+        Parameters
+        ----------
+        item : image-like object
+            Can be any of lsst.afw.image.Exposure,
+            lsst.afw.image.DecoratedImage, lsst.afw.image.Image
+            or lsst.afw.image.MaskedImage
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:209]: You've described the type. Here, you need to say _what_ it is.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 3]
```diff
@@ -200,6 +200,16 @@ def std_raw(self, item, dataId):
...
+
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:210]: Remove extra line.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 4]
```diff
@@ -200,6 +200,16 @@ def std_raw(self, item, dataId):
...
+        dataId : `dict`
+            Dataset identifier
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:212]: Needs `Returns`.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 5]
```diff
@@ -216,6 +226,22 @@ def std_raw(self, item, dataId):
...
+        """Fixup raw header metadata problems.
+
+        This should be done by the CameraMapper base class, but it isn't yet (DM-23959).
+
+        Parameters
+        ----------
+        item : `lsst.daf.base.propertyContainer.propertyList.PropertyList`
+
+        dataId : `dict`
+            Dataset identifier
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:239]: Needs `Returns`.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 6]
```diff
@@ -216,6 +226,22 @@ def std_raw(self, item, dataId):
...
 
+    def std_raw_md(self, item, dataId):
+        """Fixup raw header metadata problems.
+
+        This should be done by the CameraMapper base class, but it isn't yet (DM-23959).
+
+        Parameters
+        ----------
+        item : `lsst.daf.base.propertyContainer.propertyList.PropertyList`
+
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:237]: Describe what `item` is.

File: python/lsst/obs/pfs/pfsMapper.py [hunk 7]
```diff
@@ -216,6 +226,22 @@ def std_raw(self, item, dataId):
...
         return exp
 
+    def std_raw_md(self, item, dataId):
+        """Fixup raw header metadata problems.
+
+        This should be done by the CameraMapper base class, but it isn't yet (DM-23959).
+
+        Parameters
+        ----------
+        item : `lsst.daf.base.propertyContainer.propertyList.PropertyList`
```
Comment (PaulPrice) [inline python/lsst/obs/pfs/pfsMapper.py:236]: Type is `lsst.daf.base.PropertyList`.

File: python/lsst/obs/pfs/pfsMapper.py (hunk 4 above)
Comment (hassanxp) [reply python/lsst/obs/pfs/pfsMapper.py:212]: Added.

File: python/lsst/obs/pfs/pfsMapper.py (hunk 6 above)
Comment (hassanxp) [reply python/lsst/obs/pfs/pfsMapper.py:242]: Done.

File: python/lsst/obs/pfs/pfsMapper.py (hunk 5 above)
Comment (hassanxp) [reply python/lsst/obs/pfs/pfsMapper.py:244]: Done.

File: python/lsst/obs/pfs/pfsMapper.py (hunk 2 above)
Comment (hassanxp) [reply python/lsst/obs/pfs/pfsMapper.py:209]: Done.

**Pull request merged**